src/algebra/order/group/defs.lean

```diff
@@ -55,15 +55,19 @@ instance ordered_comm_group.to_ordered_cancel_comm_monoid [ordered_comm_group α
 example (α : Type u) [ordered_add_comm_group α] : covariant_class α α (swap (+)) (<) :=
 add_right_cancel_semigroup.covariant_swap_add_lt_of_covariant_swap_add_le α
 
+-- Backporting note: this instance is not used,
```

Comment:
Are you sure? I think the point is that the presence of this instance removes `classical.choice` from the axioms a downstream lemma. The downstream lemma will still build without it, but the proof will use choice.

Reply:
These were added in #17564

Reply:
My vote is that given this change is on the `reenableeta` branch, which we critically want to be able to merge, if there an issue that is only relevant in nonclassical mathematics, and it is not caught by building mathlib, then we *really* don't care.